app/views/shared/js/utils.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+var Utils = {
+  importScript: function(src) {
+    var script = document.createElement('script');
+    script.src = src;
+    document.head.appendChild(script);
+
+    return new Promise(function(resolve, reject) {
+      script.addEventListener('load', resolve);
+      script.addEventListener('error', reject);
+    });
+  }
+}
```

Comment:
Let's keep this code into startup.js for now. One day we will have https://bugzilla.mozilla.org/show_bug.cgi?id=568953 and it will be unneeded.